src/pages/UserProfilePage/UserProfilePage.tsx

```diff
@@ -32,6 +40,24 @@ const UserProfilePage: React.FC = () => {
     });
   }, [id, navigate]);
 
+  const handleAddFriend = () => {
+    createFriendRequest(loggedUserId!, Number.parseInt(id ?? '')).then(() => {
+      toast.success(`Successfully sent friend request to ${user?.username}!`);
+    });
+  };
+
+  useEffect(() => {
+    getFriendRequest(loggedUserId ?? 0, user?.id ?? 0)
```

Comment:
What happens here if we have 0 user in database?

Reply:
I don't think it can be 0 with the way it's implemented, but I'm not sure. What is a better solution?